### PR TITLE
Strider clutchmother Waypoints

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6818
- https://github.com/chromiecraft/chromiecraft/issues/990#issuecomment-872209992

## Tests Performed:

- Tested
- Arch Linux

## How to Test the Changes:

1.Teleport to GUID 37385
2. movment and 2 Foreststrider Fledgling

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1632387086658999212.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632387086658999212');
+
+SET @NPC := 37385;
+SET @PATH := @NPC * 10;
+
+UPDATE `creature` SET `MovementType`=2 WHERE `guid`=@NPC;
+DELETE FROM `creature` WHERE `guid` = 205803 AND `id` = @NPC;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(205803, 2321, 1, 0, 0, 1, 1, 1042, 0, 4520.04, 661.485, 24.6147, 4.93543, 275, 0, 0, 222, 0, 0, 0, 0, 0, '', 0);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1632387086658999212.sql:9]: 2321 as id is correct? should be @NPC ?

Comment (IntelligentQuantum) [reply data/sql/updates/pending_db_world/rev_1632387086658999212.sql:9]: Yeah id is correct

**Comment by syssneck** [comment]
Tested, works fine for me! They are so cute together :D
grafik

**Review by syssneck — APPROVED** [review]

**Review by Helias — APPROVED** [review]

**Comment by Hacki95** [comment]
@IntelligentQuantum Build errors

**Comment by IntelligentQuantum** [comment]
> @IntelligentQuantum Build errors

I do not know what the problem is

**Comment by IntelligentQuantum** [comment]
> @IntelligentQuantum

I do not know how to solve this problem.

File: data/sql/updates/pending_db_world/rev_1632387086658999212.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+(@PATH, 27, 4513.71, 706.448, 23.2624, 0),
+(@PATH, 28, 4528.3, 704.51, 24.6299, 0),
+(@PATH, 29, 4535.2, 690.152, 25.2516, 4000),
+(@PATH, 30, 4520.51, 660.629, 24.7118, 0);
+
+DELETE FROM `creature_formations` WHERE `leaderGUID`=@NPC;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
+(@NPC, @NPC, 0, 0, 2, 0, 0),
+(@NPC, 36692, 2, 120, 2, 0, 0),
+(@NPC, 205803, 2, 210, 0, 0, 0);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1632387086658999212.sql:52]:
```suggestion
(@NPC, @NPC, 0, 0, 2, 0, 0),
(@NPC, 36692, 2, 120, 514, 0, 0),
(@NPC, 205803, 2, 210, 514, 0, 0);
```

This should fix the issues

**Comment by IntelligentQuantum** [comment]
@Nefertumm Ty

**Pull request merged**